src/frontend-samples/explode-sample/ExplodeTile.ts

```diff
@@ -0,0 +1,629 @@
+    return this._rootTile.selectTiles(args);
+  }
+
+  /** Produce graphics of appropriate resolution to be drawn in a [[Viewport]]. */
+  public draw(args: TileDrawArgs): void {
+    assert(args instanceof ExplodeTileDrawArgs);
+
+    const tiles = this.selectTiles(args);
+    for (const tile of tiles)
+      tile.drawGraphics(args);
+
+    // Unused by this sample but helpful for debugging the tiles.
+    const rangeGfx = this.rootTile.getRangeGraphic(args.context);
+    if (undefined !== rangeGfx)
+      args.graphics.add(rangeGfx);
+
+    args.drawGraphics();
+  }
+
+  public prune(): void {
+    const olderThan = BeTimePoint.now().minus(this.expirationTime);
+    this._rootTile.prune(olderThan);
+  }
+
+  public forcePrune(): void { }
+}
+
+class RootTile extends Tile implements FeatureAppearanceProvider {
+  private readonly _elements: ElementTile[];
+  private _centerOfMass: Point3d;
+
+  public get appearanceProvider(): FeatureAppearanceProvider {
+    return this;
+  }
+
+  public constructor(tree: ExplodeTileTree, data: ElementData[], versionInfo: TileVersionInfo) {
+    const range = getRangeUnion(data.map((ele) => ele.boundingBox));
+    super({
+      isLeaf: false,
+      contentId: `${tree.id}_Root`,
+      range,
+      maximumSize: 512,
+    }, tree);
+
+    // Create tiles for each element.
+    this._centerOfMass = this.range.center.clone();
+
+    this._elements = [];
+    for (const element of data) {
+      const tileParams: TileParams = {
+        centerOfMass: this._centerOfMass,
+        versionInfo,
+        data: element,
+      };
+      const tile = new ElementTile(this, tileParams);
+      this._elements.push(tile);
+      // The range of a child element must be included in it's parent.
+      this.range.extendRange(tile.range);
+    }
+
+    this.loadChildren(); // initially empty.
+    assert(undefined !== this.children);
+
+    this.setIsReady();
+  }
+
+  /** Returns returns it's children elements that contain the graphics for drawing. */
+  public selectTiles(args: TileDrawArgs): Tile[] {
+    const selected: Tile[] = [];
+    for (const child of this._elements) {
+      const graphicsTile = child.selectTile(args);
+      if (graphicsTile)
+        selected.push(graphicsTile);
+    }
+    return selected;
+  }
+
+  /** Required by FeatureAppearanceProvider. Returns a FeatureAppearance based on element id and other factors or undefined if the it's not to be rendered. */
+  public getFeatureAppearance(source: FeatureAppearanceSource, elemLo: number, elemHi: number, subcatLo: number, subcatHi: number, geomClass: GeometryClass, modelLo: number, modelHi: number, type: BatchType, animationNodeId: number): FeatureAppearance | undefined {
+    const alwaysDrawIds = new Id64.Uint32Set(this._elements.map((element) => element.data.elementId));
+    // If the ids is one of the elements in the tree, always draw it.
+    if (alwaysDrawIds.has(elemLo, elemHi))
+      return FeatureAppearance.fromJSON({});
```

Comment:
This would be the place to apply the "emphasized" appearance if you keep that option and it's been selected in UI.